pkg/compass/app_path.go

```diff
@@ -0,0 +1,46 @@
+package compass
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// The HABITAT_APP_PATH env variable works similar to the PATH env variable, it gives a
```

Comment:
should we move this into a readme somewhere?

Reply:
I think I'll do a bigger documentation sweep later on